.stylelintrc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    "stylelint-config-standard"
+  ],
+  "fix": true,
+  "formatter": "verbose",
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "rules": {
+    "selector-list-comma-newline-after": "never-multi-line",
+    "selector-type-case": "lower",
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "declaration-empty-line-before": "never",
+    "rule-empty-line-before": [ "always-multi-line", {
+      "except": ["inside-block"],
+      "ignore": ["after-comment"]
+    }],
+    "no-descending-specificity": null,
+    "declaration-block-no-duplicate-properties": null,
+    "declaration-colon-space-after": "always-single-line",
+    "at-rule-whitelist": ["include", "import", "keyframes", "mixin", "extend"],
+    "string-quotes": "double",
+    "no-duplicate-selectors": true,
+    "color-named": "never",
```

Comment:
@h0m1c1de please check this and discuss with @alexgul4enko 
`black` and `white` is allowed in upcoming PR to codestyle repo